code/modules/mob/living/simple_mob/subtypes/occult/living_statue.dm

```diff
@@ -98,8 +109,8 @@
 	override_base = "grey"
 	hud_state = "blackout"
 
-	cooldown_min = 1 MINUTE // Overkill but by request.
 	charge_max = 300
+	cooldown_min = 300
```

Comment:
going to point this out, but I also recognize that it might be intentional
300 is 30 seconds, not 5 minutes